File: app/src/main/java/net/mbonnin/arcanetracker/hsreplay/model/UploadRequest.java
package net.mbonnin.arcanetracker.hsreplay.model;

/**
 * Created by martin on 11/29/16.
 */

public class UploadRequest {
    public String match_start;
    public String friendly_player_id;
    public int build;
    public int game_type;
    public HSPlayer player1 = new HSPlayer();
    public HSPlayer player2 = new HSPlayer();
}

File: app/src/main/java/net/mbonnin/arcanetracker/hsreplay/model/UploadRequest.kt
package net.mbonnin.arcanetracker.hsreplay.model

import net.mbonnin.arcanetracker.BnetGameType
import net.mbonnin.arcanetracker.FormatType


class UploadRequest {
    var match_start: String? = null
    var friendly_player: String? = null
    var build: Int = 0
    var game_type = BnetGameType.BGT_UNKNOWN.intValue
    var player1 = HSPlayer()
    var player2 = HSPlayer()
    var spectator_mode: Boolean = false
    var format = FormatType.FT_UNKNOWN.intValue
}
